app/views/physical_storage/show_list.html.haml

```diff
@@ -0,0 +1,7 @@
+#main_div
+  = render :partial => 'layouts/gtl'
+
+  %physical-switch-toolbar#physical_storage_show_list_form
+
+:javascript
+  miq_bootstrap('#physical_storage_show_list_form');
```

Comment:
pls, add the missing EOL (and update you editor setting to do that for you automatically)

also above

Reply:
@martinpovolny Thanks for the reminder, just added here.